src/AguaQuente.java

```diff
@@ -0,0 +1,17 @@
+
+public class AguaQuente extends NivelDeAcucar {
```

Comment:
AguaQuente é um NivelDeAcucar? Possivelmente vc utilizou herança para reaproveitar código, dê preferência a composição/agregação quando o objetivo só for reaproveitar código. Ainda assim, lembra que essa opção deveria ignorar os níveis de açúcar?